internal/controller/standalone_pgadmin/pod.go

```diff
@@ -352,17 +360,36 @@ with open('` + configMountPath + `/` + configFilePath + `') as _f:
 if os.path.isfile('` + ldapPasswordAbsolutePath + `'):
     with open('` + ldapPasswordAbsolutePath + `') as _f:
         LDAP_BIND_PASSWORD = _f.read()
+`
+		// gunicorn reads from the `/etc/pgadmin/gunicorn_config.py` file during startup
+		// after all other config files.
+		// - https://docs.gunicorn.org/en/latest/configure.html#configuration-file
+		//
+		// This command writes a script in `/etc/pgadmin/gunicorn_config.py` that reads
+		// from the `gunicorn-config.json` file and sets those variables globally.
+		// That way those values are available as settings when gunicorn starts.
+		//
+		// Note: All pgAdmin settings are lowercase with underscores, so ignore
```

Comment:
Should this say "All gunicorn settings are lowercase..."?

Reply:
Yep, I'll make that change.